src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -159,55 +145,6 @@ private static void SaveFileConfig(FileConfig fileConfig, ILogger logger)
             return defaults;
         }
 
-        internal /* for testing */ static void CallClangAnalyzer(Action<Message> handleMessage, IRequest request, IProcessRunner runner, ILogger logger, CancellationToken cancellationToken)
```

Comment:
This is the method that actually invoked the sub-process exe. It's a completely standalone method and was only called by the `CLangAnalyzer`, so I've moved it to `CLangAnalyzer`.

That should mean that the only methods left in `CFamilyHelper` are ones that relate to creating the vcxproj `Request` i.e. it should now have a single responsibility. 
I might change this into a non-static class, possibly only that implements some kind of `IRequestBuilder` builder. Either way, the helper class is simpler.